components/ILIAS/Authentication/classes/Setup/class.ilAuthenticationDatabaseUpdateSteps8.php

```diff
@@ -38,4 +38,14 @@ public function step_1(): void
             );
         }
     }
+
+    public function step_2(): void
+    {
+        if ($this->db->tableExists('settings')) {
+            $query = 'DELETE FROM settings WHERE module = ' . $this->db->quote('login_settings', 'text')
+                . ' AND ' . $this->db->like('keyword', 'text', 'login_message_%');
```

Comment:
Use `ilDBConstants::T_TEXT` instead of `text`.

Reply:
ok